contributing/workflow.md

```diff
@@ -179,6 +179,21 @@ For instance, to commit your work from a debugging session:
 
 Just make sure that your commits in a feature branch are all related.
 
+### Signing GPG Old Commits
+
+Almost developer forgot signing GPG with params `-S` in commiting like `git commit -S -m "Signed GPG"`, you can signing old commit to keep secure.
+
+```console
+> git switch your-branch
+> git commit --amend --no-edit -n -S
+> git push --force-with-lease origin your-branch
+```
```

Comment:
This seems good, but what if we have more than one commit?

Reply:
Updated if more than one commit